src/content/posts/weekly-2021-01-14.adoc

```diff
@@ -0,0 +1,38 @@
+= This Week in JBoss: 25 December 2020
+Romain Pelisse
+v1.0, 2021-01-14
+:tags:
+
+Welcome to the first installment of our JBoss Editorial for 2021! And to start the year, we have a brand new major release of the core product of the JBoss community! Yes, Wildfly 22 is out!
+
+== Wildfly 22 and Camel 3.7
+
+Yes! We have a brand new major version of Wildfly! link:https://www.wildfly.org/news/2021/01/13/WildFly22-Final-Released/[Wildfly 22 is out!].
+
+This release includes support for use of log4j2 in applications, new base health and metrics subsystems to provide server observability for users who don't need the application instrumentation that comes with MicroProfile Health and Metrics, and a number of other features related to management, provisioning, messaging and security. Plus numerous bug fixes, enhancements and component upgrades.
+
+On top of that, there is also an updated version of the new tech-preview WildFly Preview distribution. This will allow the community have a look at what Wildfly developers are doing for Jakarta EE 9 support, along with other changes in the server architecture that are expected to appear later in 2021.
+
+Wildfly 22 is already a nice treat, but we also have link:https://camel.apache.org/blog/2020/12/Camel37-Whatsnew/[Camel 3.7], which also quite exciting, and fortunately for all of us, Claus Ibsen has published a review of what he called link:http://www.davsclaus.com/2020/12/apache-camel-37-lts-released-fastest.html[the fastest Camel ever!] So, go check it out if you want to know more about this camel release.
+
+== Tech bytes
+
+A few interesting technical articles came out in the last weeks, so we'll try to catch you up with three of them here. First, we have an excellent one from Bela Ban on link:http://belaban.blogspot.com/2020/12/running-jgroups-raft-as-service.html[running JGroups raft as a service]. It's very hands on post, focusing on getting everything running. Then we have a longer walkthrough to let you experiment on link:https://blog.ramon-gordillo.dev/2020/12/spring-boot-embedded-cache-with-infinispan-in-kubernetes/[using Spring boot embedded cache with Infinispan on Kubernetes]. Certainly, worth a look, isn't it? The last one is, of course, on Quarkus, because Quarkus is, as it should, all the rage laterly. In this article, Clément Escoffier took the time to explain some of the link:https://quarkus.io/blog/magic-control[magic beneath Quarkus and Vert.x] and why this is the kind magic you to have as a developer.
```

Comment:
```suggestion
A few interesting technical articles came out in the last weeks, so we'll try to catch you up with three of them here. First, we have an excellent one from Bela Ban on link:http://belaban.blogspot.com/2020/12/running-jgroups-raft-as-service.html[running JGroups raft as a service]. It's very hands on post, focusing on getting everything running. Then we have a longer walkthrough to let you experiment on link:https://blog.ramon-gordillo.dev/2020/12/spring-boot-embedded-cache-with-infinispan-in-kubernetes/[using Spring boot embedded cache with Infinispan on Kubernetes]. Certainly, worth a look, isn't it? The last one is, of course, on Quarkus, because Quarkus is, as it should, all the rage lately. In this article, Clément Escoffier took the time to explain some of the link:https://quarkus.io/blog/magic-control[magic beneath Quarkus and Vert.x] and why this is the kind magic you to have as a developer.
```